packages/editor/src/components/post-author/constants.js

```diff
@@ -0,0 +1,6 @@
+export const AUTHORS_QUERY = {
+	who: 'authors',
+	per_page: 50,
```

Comment:
I changed the "per_page" argument from 100 to 50. However, I don't think this number makes a big difference when used with `ComboboxControl.`

Happy to revert it if needed.